airflow/cli/commands/dag_command.py

```diff
@@ -104,16 +104,15 @@ def dag_trigger(args):
     :return:
     """
     api_client = get_current_api_client()
-    log = LoggingMixin().log
     try:
         message = api_client.trigger_dag(dag_id=args.dag_id,
                                          run_id=args.run_id,
                                          conf=args.conf,
                                          execution_date=args.exec_date)
+        print(message)
     except OSError as err:
-        log.error(err)
+        print(str(err))
         raise AirflowException(err)
```

Comment:
Do we need print here? The same exception with message will be rise.

Reply:
Fixed. Raising exceptions is also not the best solution, but it is another problem.